model.go

```diff
@@ -37,6 +39,38 @@ func LoadModel(exportDir string, tags []string, options *tf.SessionOptions) (mod
 	return
 }
 
+// ImportModel creates a new *Model, loading the graph from the serialized representation.
+// This operation creates a session with specified `options`
+// Panics if the model can't be loaded
+func ImportModel(serializedModel string, options *tf.SessionOptions) (model *Model) {
+	var err error
+	model = new(Model)
+
+	defer func() {
+		if err != nil {
+			panic(err.Error())
+		}
+	}()
+
+	contents, err := ioutil.ReadFile(serializedModel)
+	if err != nil {
+		return
+	}
+
+	graph := tf.NewGraph()
+	if err := graph.Import(contents, ""); err != nil {
```

Comment:
Can you please add a parameter `prefix` to the function and pass it to the `graph.Import` call instead of `""`? In this way, we add the support to the loading of multiple identical graphs that can be placed under different prefixes.

Reply:
Done